.travis.yml

```diff
@@ -9,15 +9,32 @@ python:
   - "3.5"
   - "pypy"
   - "pypy3"
+matrix:
+  include:
+    - python: "2.7"
+      env: TOXENV=perf
 
-install:
-  - pip install -e .
-script:
-  - py.test
+before_install: |
+  pip install tox "virtualenv<14.0.0" # recent versions don't run on Python 3.2
```

Comment:
I'd rather drop 3.2 support already, as pip/virtualenv/coverage.py/hypothesis/... did as well. 

I know you argued against it in https://github.com/eisensheng/pytest-catchlog/pull/31#issuecomment-168746387 - but first 2.6 is less problematic, and I think it's in much more widespread use. Old distributions are going to be on 2.6/2.7, but if something is new enough to have Python 3, it'd at least have 3.3 (like Ubuntu Precise, the oldest still maintained Ubuntu LTS).

I'd rather avoid cumulating more and more technical dept for something probably not used by anyone.